server/planning/agendas_async/agendas_async_service.py

```diff
@@ -1,26 +1,22 @@
 from typing import Any
 from planning.core.service import BasePlanningAsyncService
+from planning.planning.planning_service import PlanningAsyncService
 from planning.types import AgendasResourceModel
 from apps.auth import get_user_id
-from superdesk import get_resource_service
+from superdesk.resource_fields import ID_FIELD
 from superdesk.errors import SuperdeskApiError
 from superdesk.notification import push_notification
 
 
-class AgendasAsyncService(BasePlanningAsyncService[AgendasResourceModel]):
-    # TODO-ASYNC: on_fetched mechanism to be added to the base REST API class
-    # async def _generate_planning_info(self, docs):
-    #     # TODO-ASYNC: Change this when get_planning_by_agenda_id is added to utils
-    #     planning_service = get_resource_service("planning")
-    #     for doc in docs:
-    #         doc["plannings"] = planning_service.get_planning_by_agenda_id(doc.get(ID_FIELD)).docs
-    #
-    # async def on_fetched(self, docs):
-    #     await self._generate_planning_info(docs.get(ITEMS))
-    #
-    # async def on_fetched_item(self, doc):
-    #     await self._generate_planning_info([doc])
+async def generate_planning_info(docs):
```

Comment:
It would be good to have types here, in this case it should be `(docs: list[dict]) -> None:`